cves/kernel/CVE-2013-0311.yml

```diff
@@ -167,8 +168,10 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  note:
-  answer:
+  note: |
+    The incorrect length variable is used in loops, so there is a chance that a
+    loop could go to long which would result in memory being incorrectly accessed
```

Comment:
I think I get your point, but it might be useful to talk specifically about how an automated tool could discover or test this.